src/views/menus/AccountMenu.tsx

```diff
@@ -66,7 +66,14 @@ export const AccountMenu = () => {
   const { usdcLabel, chainTokenLabel } = useTokenConfigs();
   const theme = useAppSelector(getAppTheme);
 
-  const { evmAddress, walletType, dydxAddress, hdKey } = useAccounts();
+  const { evmAddress, solAddress, walletType, dydxAddress, hdKey } = useAccounts();
+
+  let address: string | undefined;
+  if (walletType === WalletType.Phantom) {
+    address = truncateAddress(solAddress, '');
+  } else {
```

Comment:
Are there any other address types that dYdX supports that are not EVM hex addresses?

Reply:
in terms of wallet connections? right now we only support EVM

Reply:
cc: @rosepuppy